src/pages/Coins/Coins.styles.ts

```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components';
+
+export const Coins = styled.div`
+	margin: ${({ theme }) => theme.spacing.spacingMedium} ${({ theme }) => theme.spacing.spacingMedium};
```

Comment:
Giving the fact that you have the spacings under 'theme.spacing.' I think it would be better if the variable 'spacingMedium' would be named just 'medium'. In the end we will have 'theme.spacing.medium'. This is just a suggestion